src/Curiosity/curiosity.tsx

```diff
@@ -1,30 +1,41 @@
-
-
 import React from 'react';
 import { useState, useEffect } from "react";
-import './curiosity.scss';
+import '../RoverPhoto/RoverPhoto.scss'
+import { RoverTopPhoto } from '../RoverPhoto/RoverTopPhoto';
 
 function Curiosity() {
+    let curiosityPhoto = [];
     const [curiosityPhotoData, setCuriosityPhotoData] = useState(null);
-    console.log(process.env.REACT_APP_NASA_API_KEY)
+    const [clickedImage, setClickedImage] = useState(null);
     useEffect(() => {
-        fetch(`https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos?sol=0&api_key=${process.env.REACT_APP_NASA_API_KEY}`)
+        fetch(`https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos?sol=2&camera=navcam&api_key=${process.env.REACT_APP_NASA_API_KEY}`)
             .then(response => response.json())
-            .then(data => setCuriosityPhotoData(data))
+            .then(data => {
+                setCuriosityPhotoData(data)
+                setClickedImage(data.photos[0])
+            }
+            )
     }, []);
 
-    if (!curiosityPhotoData) {
+    if (!curiosityPhotoData || !clickedImage) {
         return <div>Waiting for data!</div>
     }
+
+    for (let i = 0; i < 6 && i < curiosityPhotoData.photos.length; i++) {
```

Comment:
I think this loop could be changed to something like `let curiosityPhotos = curiosityPhotoData.slice(0, 6)`